src/ooxml/java/org/apache/poi/xddf/usermodel/chart/XDDFBarChartData.java

```diff
@@ -34,12 +35,27 @@ Licensed to the Apache Software Foundation (ASF) under one or more
     public XDDFBarChartData(CTBarChart chart, Map<Long, XDDFChartAxis> categories,
             Map<Long, XDDFValueAxis> values) {
         this.chart = chart;
+    	if (chart.getBarDir() == null) {
```

Comment:
could you remove the tabs from your code?

Reply:
Thanks for notifying about my IDE running misconfigured!